specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/keys.json

```diff
@@ -975,6 +1028,120 @@
     }
   },
   "definitions": {
+    "KeyReleaseCondition": {
+      "properties": {
+        "claim": {
+          "x-ms-client-name": "claim name",
+          "type": "string",
+          "minLength": 1,
+          "description": "claim name"
+        },
+        "condition": {
+          "x-ms-client-name": "condition to test",
+          "type": "string",
+          "minLength": 1,
+          "description": "condition to test",
+          "enum": [
+            "equals"
+          ],
+          "x-ms-enum": {
+            "name": "KeyReleaseCondition",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "equals",
+                "description": "equals comparison."
+              }
+            ]
+          }
+        },
+        "value": {
+          "type": "string"
+        }
+      }
+    },
+    "KeyReleaseAuthority": {
+      "properties": {
+        "authority": {
+          "x-ms-client-name": "authority name",
+          "type": "string",
+          "minLength": 1,
+          "description": "authority name"
+        },
+        "allOf": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KeyReleaseCondition",
+            "description": "Required conditions."
+          }
+        }
+      }
+    },
+    "KeyReleasePolicy": {
+      "properties": {
+        "version": {
+          "x-ms-client-name": "version",
+          "description": "authority name",
+          "type": "string",
+          "enum": [
+            "0.2"
+          ]
+        },
+        "anyOf": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KeyReleaseAuthority",
+            "description": "Authorities."
+          }
+        }
+      }
+    },
+    "ProtectedKey": {
+      "properties": {
+        "alg": {
+          "x-ms-client-name": "key wrapping algorithm",
+          "type": "string",
+          "minLength": 1,
+          "description": "key wrapping algorithm identifier",
+          "enum": [
+            "dir"
+          ],
+          "x-ms-enum": {
+            "name": "ProtectedKeyWrappingAlgorithm",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "dir",
+                "description": "Direct mode."
+              }
+            ]
+          }
+        },
+        "enc": {
+          "x-ms-client-name": "encryption algorithm",
+          "type": "string",
+          "minLength": 1,
+          "description": "encryption algorithm identifier",
+          "enum": [
+            "CKM_RSA_AES_KEY_WRAP"
+          ],
+          "x-ms-enum": {
+            "name": "ProtectedKeyEncryptionAlgorithm",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "CKM_RSA_AES_KEY_WRAP",
+                "description": "RSA AES Key Wrap."
+              }
+            ]
+          }
+        },
+        "ciphertext": {
```

Comment:
Minor: This seems to be not used. Please consider removing this.

Reply:
What do you think is not used, it is not clear from your comment?

Reply:
Sorry for this. When I converted this json to yaml in **swagger editor** it showed up as misaligned. Thought that this is an unused definition.  Removing this comment.